README.md

```diff
@@ -16,14 +16,19 @@ with over the wire protocols (ex. http1/2/3 or WebSockets).
 Topics which this team discussed or challenges this team decides to take on should fall under one of the goals of the team:
 
 - Improve Node.js core api support for frameworks
+- Provide feedback and collaboration on Node.js core features and proposals
 - Improve collaboration among framework authors
-- Provide support for framework when security and bugs arise
+- Provide support for frameworks when security issues or bugs arise
+- Be a resource for Node.js security concerns in APIs which touch web frameworks (ex. HTTP1/2/3)
+- Collaborate with other Node.js WGs and teams where it makes sense
 
 Things which should not be done in this repo:
 
 - Framework wars
 - Promotion of specific tools/resource/etc. without a clear topic for collaboration/discussion
 - Putting down projects, authors or solutions to a problem
+- Determine what is/is not a framework
```

Comment:
I agree we should **not** define what is/what isn't a framework, but in that case shouldn't we remove the **What are Web-Server Frameworks** section above? That section is literally defining it...

Reply:
Haha, yes it is.  So here is the conundrum, how do we set some constraints on the scope so that folks new to the team or who randomly land on the `README.md` know a bit about what we focus on.  Is it enough to have just the intro paragraph?

Reply:
Ok, removed that section.